package.json

```diff
@@ -49,8 +49,8 @@
     "oclif": "^3.4.3",
     "postinstall-postinstall": "^2.1.0",
     "prettier": "^3.0.0",
+    "prettier-2": "npm:prettier@^2",
```

Comment:
Why would we need 2 different versions of prettier?

Reply:
Just read your comment - I would prefer to avoid this. What if we just downgrade to prettier@2 instead?

Reply:
Because Jest currently does not yet support Prettier v3 (it dropped `sync` support which Jest needs): https://jestjs.io/docs/configuration#prettierpath-string & https://github.com/jestjs/jest/issues/14305

The alternative is using Babel to format snapshot testing but that's not as nice as it doesn't handle indenting and all things considered this is a pretty small dev dependency so I figured I'd just use that 🤷

Reply:
Sure downgrading to v2 should work

Reply:
@marcolink I've opened #309 applying `prettier` to the codebase as there are a couple of files that require changes - it would be good if we can get that landed ahead of this one